source/renderer/app/components/wallet/settings/ChangeSpendingPasswordDialog.scss

```diff
@@ -4,6 +4,12 @@
 .dialog {
   font-family: var(--font-light);
 
+  :global {
+    .Dialog_contentWrapper::-webkit-scrollbar-track {
```

Comment:
@aleksandardjordjeviciohk shouldn't we put this into an override for the Dialog instead of the global here? 🤔

Reply:
This is obsolete now, as it was a leftover from previous implementation. Deleted now.